.circleci/config.yml

```diff
@@ -83,13 +91,54 @@ jobs:
           command: |
             docker push circleci/circleci-inferconfig:0.$CIRCLE_BUILD_NUM
             docker push circleci/circleci-inferconfig:latest
+  vulnerability-scan:
+    executor: go
+    steps:
+      - checkout
+      - run:
+          name: Setup Scanning
+          command: |
+              git config --global url."https://$GITHUB_USER:$GITHUB_TOKEN@github.com/circleci/".insteadOf "https://github.com/circleci/"
```

Comment:
Is this step still needed for a public repo like this (with no private libs being used)?

Reply:
I don't think so. I'll remove this step.